ynh_add_swap/ynh_add_swap

```diff
@@ -1,8 +1,16 @@
 #!/bin/bash
 
+# Need also the helper https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/ynh_handle_getopts_args/ynh_handle_getopts_args
+
 # Argument $1 is the size of the swap in MiB
```

Comment:
Did you forgot to replace `$1` ?

Reply:
Not really, it's more a matter of structured comment missing here. As you can see.